themes/cerberus/index.txt

```diff
@@ -1,31 +1,39 @@
-<%- title %>
-
+<%- title %><br />
+<br />
 <% if (locals.intro) { %>
 <% intro.forEach(function (introItem) { -%>
-<%- introItem %>
+<%- introItem %><br />
 <% }) -%>
 <% } %>
-
+<% if (locals.intro) { %>
```

Comment:
Same goes for `locals.dictionary`, `locals.action` and `locals.outro`. Also applies to other themes.